schemas/list-worker-types-response.yml

```diff
@@ -0,0 +1,7 @@
+---
+  id: "http://schemas.taskcluster.net/aws-provisioner/v1/list-worker-types-response.json#"
+  title: "List Worker Types"
+  type: "array"
+  items: 
+    type: "string"
```

Comment:
`pattern: '^[a-zA-Z0-9-_]{1,22}$'` to ensure that they are valid identifiers.

In taskcluster we use slugid which matches `/^[a-zA-Z0-9-_]{22}$/` and human readable identifiers for a lot of other things, these matches `/^[a-zA-Z0-9-_]{1,22}$/`. Small is powerful when we need to stick into SSH keynames (on EC2), azure queue names, rabbitmq routingkeys, urls, and so forth...

The choice may seem a bit arbitrary, but it's a fairly well support character set... It's small, and can be decoded as URL-safe base64, if one wants to encode it efficiently in another format, for example base32.